### ISSUE TITLE
mobilenetv2 + detection  training problems

### ISSUE THREAD

**Issue by jacky4323** [liangfu/mxnet-mobilenet-v2#5]
thanks for your great work!!

I have some questions.
I use this model(multiplier = 1.0)  to train my detection model and I have to resize my input probably to 
300x300 or 512x512 or 416x416,but the pretrained model you provided is 224x224,If this will cause some problems to train detection model ?

thanks for your suggestions.

**Comment by liangfu** [comment]
I have trained a ssd model with the features. Even the input image size is as large as 512x1024, the pretrained model works fine.

**Comment by liangfu** [comment]
i my observation, there is no need to fix the parameters in mobilenetv2 while training ssd.

**Comment by jacky4323** [comment]
Hi , @liangfu 

I  have train on pascol voc dataset , it just can achieve 68 mAP , what is the limit of this model?(75%、80%?)
thanks!!

**Comment by liangfu** [comment]
can you provide more detail on how did u train your model? and what exactly do you mean by referring to the limit of the model?

**Comment by jacky4323** [comment]
I am still training now , I'm not very rigorous to tune the parameters, so after training complete I will back :)
I also notice that mxnet gluon model zoo have some pretrained model,Could I download the params file and use your symbol to train?
or I still have to rename the layers' name due to different name for pretrained model

thank you for your patience :)

**Comment by jacky4323** [comment]
I have downloaded the models in the models folder. in the begining,the performance isn't good at PacolVoc,now I strictly tune the parameters and still training the model now so I don't know the performance.
gluon model zoo also has multiplier = 0.5 0.75 I am interested(due to less computation)
thanks!!

**Comment by liangfu** [comment]
that make sense. i can upload pretrained model with multiplier with 0.75 and 0.5 then, so that more people could benefit from this.

**Comment by jacky4323** [comment]
Hi @liangfu 

thanks for your kindness, I think it will be very helpful !!

**Comment by jacky4323** [comment]
@liangfu  Hi,

I'm back :)
I use https://github.com/zhreshold/mxnet-ssd to train mobilenetv2 model 
but in input  data shape 300x300 and initialize learning rate 0.002 and  scale 0.1 it at epoch 80 and 160
the experiment result at PascolVoc just can achieve 60 mAP , is that resonable? or Could this model achieve 75 mAP?

Thanks!

**Comment by liangfu** [comment]
I think mobilenetv2 based ssdlite should be reproducible if you stick on the details described in the paper.

Can you tell which layers did you feed into the detection layers? I think they supposed to be the element-wise added shortcut layers.

**Comment by jacky4323** [comment]
Hi, 

seq-5-block0-exp-batchnorm、last-1x1-conv-batchnorm and four extra layers(all these layers is depthwise
followed by 1 × 1 projection) to feed into detection layers(prediction confidence and location use normal convolution)

![image](https://user-images.githubusercontent.com/31586393/43695124-509a81e2-9969-11e8-9685-d8bd9d588a29.png)


What is the meaning of reproducible ? I only train on PascolVoc and I didn't knowwhat performance it can achieve(70 mAP or 75 mAP or 80 mAP) 

thank you!!

**Comment by liangfu** [comment]
I mean to reproduce the mAP results stated in the paper.

IMHO, "the last layer" you highlighted means the shortcut layers in mobilenetv2, not the batchnorm layer.
